backend/document-service/src/main/kotlin/com/nosql/document/converter/DocumentDtoToDocumentEntityConverter.kt

```diff
@@ -0,0 +1,18 @@
+package com.nosql.document.converter
+
+import com.nosql.document.dto.DocumentDto
+import com.nosql.document.entity.DocumentEntity
+import org.bson.types.ObjectId
+import org.springframework.core.convert.converter.Converter
+import org.springframework.stereotype.Component
+
+@Component
+class DocumentDtoToDocumentEntityConverter : Converter<DocumentDto, DocumentEntity> {
+
+    override fun convert(source: DocumentDto) = DocumentEntity(
+        id = if (source.id == null) null else ObjectId(source.id),
```

Comment:
```suggestion
        id = source.id?.let { ObjectId(it) }
```
Maybe it would be better?

Reply:
fixed